src/model/common-interfaces.ts

```diff
@@ -36,7 +36,7 @@ export interface NgcLayouts {
  * Interface representing the cookie consent status.
  */
 export interface NgcCookieConsentStatus {
-  [key: string]: 'deny' | 'allow' | 'dismiss';
+  [key: string]: 'deny' | 'allow' | 'dismiss' | undefined;
```

Comment:
this line could be removed, i don't see why it was even there in the first place, as the possible values are already listed below.
```suggestion
```